Sources/ContentView.swift

```diff
@@ -5,25 +5,32 @@ struct ContentView: View {
     @State
     var repositories = [Repository]()
     @State
-    private var searchTerm: String = "php"
+    private var searchTerm: String = ""
+    @State
+    private var isError = false
+    @State
+    private var message = ""
 
     var body: some View {
         NavigationView {
-            VStack {
-                List {
-                    ForEach(repositories) { repository in
-                        Text(repository.name ?? "Pero")
-                    }
+            List {
+                ForEach(repositories) { repository in
+                    Text(repository.name ?? "N/A")
```

Comment:
Just a personal preference, I would put lowercased "n/a" instead of uppercased "N/A". Also I think lowercased is more common in codebases.

_n/a is a common abbreviation in tables and lists for the phrase not applicable, not available, not assessed, or no answer._